### PR TITLE
Feature/create an integration test for creating wallet

### PR DESCRIPTION
**Pull request by DawidStuzynski**

### CODE REVIEW

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class WalletTestIT extends BaseControllerTestIT {
+
+    @Autowired
+    private WalletRepository walletRepository;
+
+    @Test
+    @DisplayName("It should create wallet")
+    void shouldCreateWallet() throws Exception {
```
Comment (odzio33) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java:21]: you have to be sure the table is empty before start test

**Review by odzio33 — CHANGES_REQUESTED** [review]
changes required

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java (hunk 1 above)
Comment (DawidStuzynski) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java:22]: Okay, done

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import pl.byczazagroda.trackexpensesappbackend.dto.CreateWalletDTO;
+import pl.byczazagroda.trackexpensesappbackend.repository.WalletRepository;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+class WalletTestIT extends BaseControllerTestIT {
+    @Autowired
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java:17]: Please add empty line before @Autowired

Comment (DawidStuzynski) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/WalletTestIT.java:17]: Done

**Review by dawciobiel — APPROVED** [review]
Looks good for me.

**Review by dawciobiel — DISMISSED** [review]
Looks good for me

**Review by odzio33 — APPROVED** [review]
Looks good for me

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java [hunk 3]
```diff
@@ -125,10 +125,21 @@ void shouldSuccessfullyFindFinancialTransaction_WhenFindingWithProperTransaction
         Assertions.assertEquals(financialTransactionDTO, foundTransaction);
     }
 
+    @Test
+    @DisplayName("when deleting financial transaction that does not exist should throw an exception")
+    void ShouldThrowAnException_WhenGivenTransactionDoesNotExist() {
+        //given
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java:131]: I'm not sure there should be an empty line between //given and //when sections. Can you verify it?

File: src/main/resources/db/changelog/data/financialTransaction.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+id,wallet_id,transaction_type,amount,transaction_date,description
+1,1,INCOME,15000,2022-02-19T09:47:52.595721658,""
```
Comment (dawciobiel) [inline src/main/resources/db/changelog/data/financialTransaction.csv:2]: What is the start/end character for data content for a single field in CSV for Liquibase data?
Is it ", ', | or else?
You know, if in the Liquibase config(?) we got another character this `""` gonna be paste just as String: `""` - not EMPTY_STRING value.

File: src/main/resources/db/changelog/data/financialTransaction.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+2,2,EXPENSES,5000,2022-02-19T09:47:52.595721658,null
```
Comment (dawciobiel) [inline src/main/resources/db/changelog/data/financialTransaction.csv:3]: How to paste in proper way `null` value in CSV file? How CSV or Liqubase config (?) know it's not just a String value "null" ?

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Please read my comments and reply to them.

**Review by dawciobiel — APPROVED** [review]
Looks good for me

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Pull request merged**